components/runtime/squeak4/squeak-4.p5m

```diff
@@ -0,0 +1,59 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 David Stes
+#
+
+
+<transform file -> add pkg.depend.bypass-generate libGL\.so\.1>
```

Comment:
Please, format manifest according to https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/doc/best-practices.md rule 10, this one is pkgfmt'd , but unreadable.